job_templates/puppet_agent_disable_-_ssh_default.erb

```diff
@@ -13,4 +13,7 @@ template_inputs:
 provider_type: SSH
 kind: job_template
 -%>
+<% if @host.operatingsystem.family == 'Debian' -%>
```

Comment:
Why is this only a problem on Debian?

Reply:
What we have found out is, that on CentOS/RH the $PATH is set even for no-login shells.